pkg/core/data/ipc/transactions/provider.go

```diff
@@ -392,3 +408,8 @@ func UMember(r *rusk.Provisioner, t *user.Member) {
 		}
 	}
 }
+
+func injectRuskVersion(ctx context.Context) context.Context {
+	md := metadata.New(map[string]string{"x-rusk-version": "0.5.0"})
```

Comment:
Could you move rusk (hard-coded) version  in `pkg/config/consts.go`?

Reply:
Done